docs/advanced/api.rst

```diff
@@ -14,12 +14,24 @@ Installation
 Fedora/CentOS
 -------------
 
-To allow remote access to the control agent REST API, and for agent connections,
+To enable Flocker control service.
```

Comment:
Elsewhere in the docs this is "the control service" so perhaps that or "the Flocker control service".